code/modules/client/preferences.dm

```diff
@@ -1133,16 +1133,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 						preferred_map = maplist[pickedmap]
 
 				if ("clientfps")

```

Comment:
someone look at this whole file's changes and make sure it's right please i'm not 100% confident